challenges/5-arraycardio/arraycardio.js

```diff
@@ -99,7 +146,12 @@ function swap(arr, index1, index2) {
     return the copy.
 */
 function copy(arr) {
-    // implementation
+    let newArr = [];
+    for (let i = 0; i < arr.length; i++) {
+        let element = arr[i];
+        newArr.push(element);
```

Comment:
this is fantastic. If you wanted to be more terse, you could just do `newArr.push(arr[i])`